### PR TITLE
Integrate swagger docs

### PR DESCRIPTION
**Pull request by AmineAfia**
Add all needed annotations and setup to serve swagger docs using openapi 2.0 spec

### CODE REVIEW

**Comment by AmineAfia** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/insight/81?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#81** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/insight/81?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#79** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/insight/79?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @AmineAfia and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: cmd/api.go [hunk 1]
```diff
@@ -21,11 +26,31 @@ var (
 	}
 )
 
+// @title Thirdweb Insight
+// @version 0.0.1
```
Comment (iuwqyir) [inline cmd/api.go:30]: v0.0.1-beta to match the github tag

**Review by iuwqyir — APPROVED** [review]

File: cmd/api.go (hunk 1 above)
Comment (AmineAfia) [reply cmd/api.go]: done

**Comment by AmineAfia** [comment]
### Merge activity

* **Oct 1, 1:29 PM EDT**: @AmineAfia merged this pull request with [Graphite](https://app.graphite.dev/github/pr/thirdweb-dev/insight/81).

**Pull request merged**